src/winetricks

```diff
@@ -12292,6 +12292,81 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
+    title="PowerShell Core" \
+    publisher="Microsoft" \
+    year="2024" \
+    media="download" \
+    file1="PowerShell-7.2.19-win-x64.msi" \
+    file2="PowerShell-7.2.19-win-x86.msi"
+
+
+# Function to download and install PowerShell Core
+load_powershell_core()
+{
+    #w_package_unsupported_win32
+
+    # Download PowerShell Core 7.2.x MSI (Latest LTS Version to support win32)
+    if [ "${W_ARCH}" = "win64" ]; then
+        # https://github.com/PowerShell/PowerShell/releases/v7.2.19
+        w_download https://github.com/PowerShell/PowerShell/releases/download/v7.2.19/PowerShell-7.2.19-win-x64.msi 965e10fc77dbce6ef622f1c0fcddc03bebd2b3629f1a3253cc17a8ffc71a6522
+        msi="PowerShell-7.2.19-win-x64.msi"
+    elif [ "${W_ARCH}" = "win32" ]; then
+        # https://github.com/PowerShell/PowerShell/releases/v7.2.19
+        w_download https://github.com/PowerShell/PowerShell/releases/download/v7.2.19/PowerShell-7.2.19-win-x86.msi cbf214d41bc8be4b1ce5ba566dbfee5f335971c48231825e90af7954356ce797
+        msi="PowerShell-7.2.19-win-x86.msi"
```

Comment:
Same.

Reply:
Fixed by 06fca563298ecfca1a77cfb6079c2e7668f5fbd4 and 82c6052d9b778457ad0391453ad9b0ce0de04fbe